azure/v11.3.2/release-notes.md

```diff
@@ -0,0 +1,18 @@
+## :zap:  Giant Swarm Release 11.3.2 for Azure :zap:
+
+**If you are upgrading from 11.3.1, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release improves the reliability of NGINX Ingress Controller.
+
+Specifically, liveness probe is configured to be more fault tolerant than readiness probe. This helps shed load when it goes beyond replica capacity, speeding up recovery when NGINX gets overloaded.
+
```

Comment:
```suggestion

**Note when upgrading from v11.2.x to v11.3.x:**

This release contains the replacement of CoreOS with Flatcar introduced in v11.3.0. Please carefully read release notes for 11.3.0 including Flatcar OS migration [steps](https://github.com/giantswarm/releases/blob/master/release-notes/azure/v11.3.0.md) .

**Note for future v11.3.x releases:**

Please persist this and the above note until all customers are in v11.3.0 and above.

```

@sslavic would this make sense?

Reply:
Yes, it might help. Merged.

Still we need to find a more reliable way of informing customers about changes included in the upgrade which can span multiple platform releases. They have to be presented with and read all the release notes in between, not just the release notes of the target release. In that case we can keep release notes for each release focused only on introduced changes.

Reply:
@sslavic I created an issue here to have broader discussion on that https://github.com/giantswarm/giantswarm/issues/11513 :)